README.md

```diff
@@ -1,6 +1,47 @@
-# API Elements
+# API Elements: JavaScript
 
-## Using Lerna
+[API Elements](https://apielements.org/) is a uniform interface for dealing
+with API Descriptions. This repository contains tooling for handling API
+Elements in JavaScript along with parsers and serializers for API Description
+languages.
+
+API Element Adapters:
+
+- [API Blueprint Parser](packages/fury-adapter-apib-parser)
+- [API Blueprint Serializer](packages/fury-adapter-apib-parser)
+- [OpenAPI 2 Parser](packages/fury-adapter-swagger)
+
+## Usage
+
+```javascript
+const { Fury } = require('fury');
+
+const fury = new Fury();
+
+// Load any parsers or serializer adapters you wise to use
+const apiBlueprintParser = require('fury-adapter-apib-parser');
+fury.use(apiBlueprintParser);
+
+const openAPI2Parser = require('fury-adapter-swagger');
+fury.use(openAPI2Parser);
+
+const source = 'FORMAT: 1A\n# My API');
```

Comment:
Perhaps a multiline string which looks like _API description_ would do a better justice to the example:

```js
const source = `
FORMAT: 1A

# My API
## GET /message
+ Response 200 (text/plain)

        Hello World!
`;
```

(the document stolen from the APIB homepage)